[BUG] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project tpcds-gen

**environment**:
ubuntu20.04
JDK17
maven 3.9.3

**steps:**
sed -i '/patch -R -p1/d' $base_dir/tpcds-gen/Makefile
cd $base_dir/tpcds-gen && make clean all LINUX_CC='gcc -fcommon'

**Error:**
[2023-07-19T01:46:18.042Z]       Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/3.5.0/plexus-utils-3.5.0.jar (267 kB at 22 MB/s)
[2023-07-19T01:46:18.042Z]       [INFO] Changes detected - recompiling the module! :dependency
[2023-07-19T01:46:18.042Z]       [WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
[2023-07-19T01:46:18.042Z]       [INFO] Compiling 1 source file with javac [debug target 1.6] to target/classes
[2023-07-19T01:46:18.042Z]       [INFO] -------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [WARNING] COMPILATION WARNING : 
[2023-07-19T01:46:18.042Z]       [INFO] -------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [WARNING] bootstrap class path not set in conjunction with -source 6
[2023-07-19T01:46:18.042Z]       [INFO] 1 warning
[2023-07-19T01:46:18.042Z]       [INFO] -------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [INFO] -------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [ERROR] COMPILATION ERROR : 
[2023-07-19T01:46:18.042Z]       [INFO] -------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [ERROR] Source option 6 is no longer supported. Use 7 or later.
[2023-07-19T01:46:18.042Z]       [ERROR] Target option 6 is no longer supported. Use 7 or later.
[2023-07-19T01:46:18.042Z]       [INFO] 2 errors 
[2023-07-19T01:46:18.042Z]       [INFO] -------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [INFO] ------------------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [INFO] BUILD FAILURE
[2023-07-19T01:46:18.042Z]       [INFO] ------------------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [INFO] Total time:  3.887 s
[2023-07-19T01:46:18.042Z]       [INFO] Finished at: 2023-07-19T01:45:18Z
[2023-07-19T01:46:18.042Z]       [INFO] ------------------------------------------------------------------------
[2023-07-19T01:46:18.042Z]       [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project tpcds-gen: Compilation failure: Compilation failure: 
[2023-07-19T01:46:18.042Z]       [ERROR] Source option 6 is no longer supported. Use 7 or later.
[2023-07-19T01:46:18.042Z]       [ERROR] Target option 6 is no longer supported. Use 7 or later.
[2023-07-19T01:46:18.042Z]       [ERROR] -> [Help 1]
[2023-07-19T01:46:18.042Z]       [ERROR] 
[2023-07-19T01:46:18.042Z]       [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[2023-07-19T01:46:18.042Z]       [ERROR] Re-run Maven using the -X switch to enable full debug logging.
[2023-07-19T01:46:18.042Z]       [ERROR] 
[2023-07-19T01:46:18.042Z]       [ERROR] For more information about the errors and possible solutions, please read the following articles:
[2023-07-19T01:46:18.042Z]       [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException

--- a/nds/tpcds-gen/pom.xml
+++ b/nds/tpcds-gen/pom.xml
@@ -30,6 +30,10 @@
   <name>tpcds-gen</name>
   <url>http://maven.apache.org</url>
 
+  <properties>
+    <tpcds-gen.jdk.version>1.8</tpcds-gen.jdk.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
@@ -62,8 +66,8 @@
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>${tpcds-gen.jdk.version}</source>
+          <target>${tpcds-gen.jdk.version}</target>
         </configuration>
       </plugin>
       <plugin>

